Automattic/wp-calypso
# Launchpad: Remove site preview domain upsell banner

Related to #

## Testing/Review Estimation

Testing: short
Review: medium

**Launchpad site preview after changes:**

<img width="1064" alt="CleanShot 2023-02-27 at 14 38 07@2x" src="https://user-images.githubusercontent.com/10482592/221665554-ec236128-59d5-4825-ac1c-3d83d689fe89.png">


**Post editor after changes (no changes here):**

![CleanShot 2023-02-27 at 14 37 20@2x](https://user-images.githubusercontent.com/10482592/221665427-ca775265-0d92-493d-a7e8-d4c0cde9cc74.png)




## Proposed Changes

* Removing domain upsell banner from Launchpad site preview. 

## Testing Instructions



* Check out this branch
* Create or use an existing launchpad enabled site
* Verify the domain upsell banner is no longer present in the site preview.
* Verify the domain upsell banner is still present in the post editor

## Pre-merge Checklist
